__tests__/app.test.js

```diff
@@ -196,4 +196,13 @@ test('Responds with an error 404 for a nonexistent id', () => {
         expect(body.error.message).toBe('Page Not Found')
     })
 })
+test('Responds with an error 401 for an invalid username', () => {
+    return request(app)
+    .post('/api/articles/4/comments')
+    .send({username: 'anon123', body: 'No idea what I am doing'})
+    .expect(401)
+    .then(({body}) => {
+        expect(body.error.message).toBe('Invalid Username')
```

Comment:
We could add a test to check that the author or body are present, if they are not we should receive a 400 🙂